src/js/helpers/Util.js

```diff
@@ -122,6 +122,19 @@ var Util = {
   isEmptyString: function (str) {
     return this.isString(str) && (str == null || str === "");
   },
+  isValidPort: function (value) {
```

Comment:
This should go as a private function into the FormValidators directly imho.
Only place where we are using this is there.